pkg/resource/namespace.go

```diff
@@ -154,6 +158,21 @@ func (ns *namespaceService) watchStatus() error {
 }
 
 func compareEvents(now []NamespaceEvent, before []NamespaceEvent) []NamespaceEvent {
+
+	var s1 []string
+	var s2 []string
+	logrus.Debug("before")
+	for _, e := range before {
+		s1 = append(s1, e.Namespace)
+	}
+	logrus.Debugf("before : %s", s1)
+
+	logrus.Debug("after")
+	for _, e := range now {
+		s2 = append(s2, e.Namespace)
+	}
+	logrus.Debugf("after : %s", s2)
```

Comment:
remove this because this was only for debugging ?

Reply:
yep.